frontend/src/component/changeRequest/ChangeRequest/Changes/Change/VariantPatch/VariantPatch.tsx

```diff
@@ -39,12 +52,39 @@ export const VariantPatch = ({
     )?.variants;
 
     return (
-        <ChangeItemCreateEditWrapper>
-            <ChangeItemInfo>
-                <Typography>Updating variants:</Typography>
-                <Diff preData={preData} data={change.payload.variants} />
-            </ChangeItemInfo>
-            {discard}
-        </ChangeItemCreateEditWrapper>
+        <ChangeItemInfo>
+            <StyledChangeHeader>
+                <TooltipLink
+                    tooltip={
+                        <Diff
+                            preData={preData}
+                            data={change.payload.variants}
+                        />
+                    }
+                    tooltipProps={{
+                        maxWidth: 500,
+                        maxHeight: 600,
+                    }}
+                >
+                    Updating variants:
```

Comment:
```suggestion
                    Updating variants to:
```
I think this makes it clear that we're showing the end result after the CR is applied.

Reply:
WDYT @NicolaeUnleash @thomasheartman ?

Reply:
Yes, I very much agree with this suggestion 🙋🏼